test/pom.xml

```diff
@@ -31,7 +31,7 @@
 
     <dependency>
       <groupId>io.zeebe</groupId>
-      <artifactId>zeebe-broker-core</artifactId>
+      <artifactId>zeebe-broker</artifactId>
```

Comment:
@spulci are we going to keep this name then?

Reply:
I think so...actually the package has been renamed on mvn central repository. To compile correctly we should use the new name until the previous name will be restored (I think it will stay with this name even in future releases)